docs/plugins.md

```diff
@@ -14,183 +14,187 @@ Looking for help using Snowpack in your project?
 
```

Comment:
NIT: Above this, the word "programmatically" is misspelled.

https://github.com/pikapkg/snowpack/pull/607/files#diff-0916b758c148ca9d7bbfb241d34d747eR10

Reply:
💯